internal/services/cdn/cdn_frontdoor_route_resource.go

```diff
@@ -326,7 +326,7 @@ func resourceCdnFrontDoorRouteRead(d *pluginsdk.ResourceData, meta interface{})
 	ctx, cancel := timeouts.ForRead(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
-	id, err := parse.FrontDoorRouteID(d.Id())
+	id, err := parse.FrontDoorRouteIDInsensitively(d.Id())
```

Comment:
this would signify the resource has been imported incorrectly, which there's validation to check this is valid, so this doesn't need to be set:

```suggestion
	id, err := parse.FrontDoorRouteID(d.Id())
```